aptos-move/framework/aptos-framework/sources/status.move

```diff
@@ -0,0 +1,53 @@
+/// This module code to assert that it is running in genesis (`Self::assert_genesis`) or after
+/// genesis (`Self::assert_operating`). These are essentially distinct states of the system. Specifically,
+/// if `Self::assert_operating` succeeds, assumptions about invariants over the global state can be made
+/// which reflect that the system has been successfully initialized.
+module aptos_framework::status {
```

Comment:
Can't this be part of genesis.move instead of its own module?

Reply:
No, unfortunately. If it's part of `genesis.move`, other modules can't do `genesis::is_operating()` due to the cyclic dependency (`genesis` uses `stake`. `stake` also need to use `genesis`.).

Reply:
Ah I see :( How about renaming it to chain_status or something?

Reply:
Yeah, we can do that. We can also merge `chain_id` and `chain_status` into `chain_info`. This seems to be a breaking change though.